docs/pyproject.md

```diff
@@ -375,6 +375,15 @@ Here, we will have the `my_package_cli` script installed which will execute the
 When a script is added or updated, run `poetry install` to make them available in the project's virtualenv.
 {{% /note %}}
 
+```toml
+[tool.poetry.scripts]
+my_executable = { reference = "some_binary.exe", type = "file" }
+```
+
+With this configuration, the file `some_binary.exe` is looked for in the directory of the active `pyproject.toml` and copied into the installation folder.
+On Windows, the `.exe` extension is added to the file and the file will be placed in the `Scripts/` directory.
+On Linux no extension is added and the destination is `bin/`.
```

Comment:
```suggestion
* On Windows, the `.exe` extension is added to the file, and is placed in the `Scripts/` directory.
* On Linux no extension is added, and is placed in the `bin/` directory.
```

For better readability, perhaps?

Reply:
This is ungrammatical.  "The '.exe' extension... is placed in the 'Scripts/' directory" is not what you intend.

Reply:
I agree the readability of this is not optimal. From the way it is currently described the full paragraph seems to imply that on Windows you'd end up with `some_file.exe.exe`

> With this configuration, the file `some_binary.exe` is looked for in the directory....On Windows, the `.exe` extension is added to the file

Perhaps something like this
> Poetry will look for the referenced file in the directory of the active project and then copied to the OS installation directory.
> * On Windows the `.exe` extension is added to the file and it is copied to the Python Scripts directory
> * On *nix systems the file will be copied to the environment `bin` directory

Reply:
Updates the text. The `.exe` bit was weird indeed, it should read clearly now.